src/Files.App/Actions/Navigation/OpenDirectoryInNewTabAction.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Copyright (c) 2023 Files Community
+// Licensed under the MIT License. See the LICENSE.
+
+namespace Files.App.Actions
+{
+	internal class OpenDirectoryInNewTabAction : ObservableObject, IAction
+	{
+		private readonly IContentPageContext context;
+
+		private readonly IUserSettingsService userSettingsService;
+
+		private readonly MainPageViewModel _mainPageViewModel;
+
+		public string Label
+			=> "OpenDirectoryInNewTab".GetLocalizedResource();
+
+		public string Description
+			=> "OpenDirectoryInNewTabDescription".GetLocalizedResource();
+
+		public RichGlyph Glyph
+			=> new(opacityStyle: "ColorIconOpenInNewTab");
+
+		public bool IsExecutable =>
+			context.HasSelection &&
+			context.SelectedItem is not null &&
+			context.SelectedItem.IsFolder &&
+			userSettingsService.GeneralSettingsService.ShowOpenInNewTab;
```

Comment:
The command is enabled only when you select one folder, we should keep the current behaviour and allow to open up to 5 of them.
You should check that `SelectedItems.Count < 5` and that all the selected items are folder

Reply:
Fixed (SelectedItems.Count <= 5)